src/pages/Detail/Detail.tsx

```diff
@@ -0,0 +1,40 @@
+import styles from './Detail.module.css';
+import DetailCard, {
+  DetailCardProps,
+} from '../../components/DetailCard/DetailCard';
+import { useEffect, useState } from 'react';
+import OverlayMenu from '../../components/OverlayMenu/OverlayMenu';
+
+function Detail() {
+  const [series, setSeries] = useState<DetailCardProps | null>(null);
+
+  const getDetails = async () => {
+    const response = await fetch('http://localhost:3001/api/details');
+    const data = await response.json();
+    setSeries(data);
+  };
+
+  // console.log(series?.genres[0].name);
```

Comment:
```suggestion
```

Reply:
Maybe I want to use this later, so I don't want to delete it completely..